packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/java/integrationsteps.md

```diff
@@ -1,3 +1,251 @@
+
+<div class="common-image-format">
+
+![Displays the Java SDK's phone authenticator enrollment form with Skip button](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-auth-list-phone-java.png)
+
+</div>
+
+### Step 8, Option 1: User selects phone authenticator
+
+In this use case option, the user selects the optional **Phone** factor as the authenticator to verify. Pass this selected authenticator to the `IDXAuthenticationWrapper.selectAuthenticator` method.
+
+```java
+authenticationResponse = idxAuthenticationWrapper.selectAuthenticator(proceedContext, authenticator);
+```
+
+The response from this request is an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_DATA`. This status indicates that the user needs to provide additional authenticator information. In the case of the phone authenticator, the user needs to specify a phone number, and whether they want to use SMS or voice as the verification method.
+
+### Step 9, Option 1: User enters phone number and selects SMS verify method
+
+You need to build a form to capture the user's phone number as well as a subsequent form for the user to select their phone verification method (either SMS or voice).
+
+<div class="common-image-format">
+
+![Displays the Java SDK's enroll phone number authenticator form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-num-java.png)
+
+</div>
+
+> **Note:** The Java SDK requires the following phone number format: `{+}{country-code}{area-code}{number}`. For example, `+15556667777`.
+
+<div class="common-image-format">
+
+![Displays the Java SDK's phone factor (SMS or voice) form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-phone-mode-java.png)
+
+</div>
+
+When the user enters their phone number and selects SMS to receive the verification code, capture this information and send it to the `IDXAuthenticationWrapper.submitPhoneAuthenticator` method.
+
+For example, the following code snippet passes the `phone` and `mode` variables to `idxAuthenticationWrapper.submitPhoneAuthenticator`:
+
+```java
+ProceedContext proceedContext = Util.getProceedContextFromSession(session);
+
+AuthenticationResponse authenticationResponse =
+   idxAuthenticationWrapper.submitPhoneAuthenticator(proceedContext,
+         phone, getFactorFromMethod(session, mode));
+```
+
+The Java SDK sends the phone authenticator data to Okta. Otka processes the request and sends an SMS code to the specified phone number. After the SMS code is sent, Okta sends a response to the SDK, which returns `AuthenticationStatus=AWAITING_AUTHENTICATOR_VERIFICATION`. This status indicates that the user needs to provide the verification code for the phone authenticator.
+
+You need to build a form to capture the user's SMS verification code. You can reuse the verification code form similar to [Step 6: User selects email authenticator](#step-6-user-selects-email-authenticator).
```

Comment:
Step 6 doesn't currently contain any code to reuse.

Reply:
I was talking about the verify-code form. I'll take this part out if it's confusing.

Reply:
Sounds good.